src/config/dynamic-theme-fixes.config

```diff
@@ -4733,6 +4733,13 @@ INVERT
 
 ================================
 
+tianchi.aliyun.com
+
+NO INVERT
+img
```

Comment:
`img` This is every image, try to just select certain images who need it.

Reply:
Thank you for your reply. And actually all `image`s showed in `tianchi.aliyun.com` have some problems, for example the words in image can't see clearly under dark mode. So I set all images.